portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/Forms.testcase

```diff
@@ -1354,7 +1354,7 @@ definition {
 			formPageName = "Page 1",
 			formTitleNumber = "1");
 
-		Form.addTextField(fieldPositionNumber = "1");
+		Form.addTextField(formPageNumber = "0");
```

Comment:
@mlan0, is this parameter necessary even with the changes introduced by the previous commit?